src/config.h

```diff
@@ -10,14 +10,19 @@
 #define VKEY_ENTITY_COUNT_UNLIMITED UINT64_MAX
 
 typedef struct {
+	int omp_thread_count;        // Maximum number of OpenMP threads, -1 indicates the default value.
```

Comment:
Default should be the value
`GxB_get (GxB_NTHREADS, int *nthreads_max) ;`
reports.

Reply:
If we don't receive an explicit `OMP_THREAD_COUNT` variable, I'd prefer to not set a default at all, so that we respect the `OMP_NUM_THREADS` environment variable if the user has one set.